lime/lime_tabular.py

```diff
@@ -316,9 +316,19 @@ def explain_instance(self,
         feature_names = copy.deepcopy(self.feature_names)
         if feature_names is None:
             feature_names = [str(x) for x in range(data_row.shape[0])]
-
-        values = self.convert_and_round(data_row)
-
+        # get column names if numpy.ndarray or pandas df:
+        try:
+            col_names = list(data_row.columns)
+        except AttributeError:
+            if data_row.dtype.names is not None:
+                col_names = list(data_row.dtype.names)
+            else:
+                col_names = []
+        if len(col_names) != 0:
+            if np.sum([1 if str(k).isdigit() else 0 for k in col_names]) != len(col_names):
```

Comment:
I don't understand this line. Why is it meaningful if column names are all digits, and why does it matter if all columns have digit names? Also, what happens if this is False? values seem to be undefined then